index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <nav class="navbar">
+      <a href="#welcome-section" class="links">About</a>
+      <a href="#project" class="links">Work</a>
+      <a href="#profile-link" class="links">Contact</a>
+    </nav>
+    <div id='welcome-section'>
+      <h1>Hey! We are Micronauts</h1>
+      <div>We are web developers</div>
+    </div>
```

Comment:
I would suggest you use section tag instead of div this will make your code SEO friendly.